src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceCreateAndUp.kt

```diff
@@ -0,0 +1,14 @@
+package com.cognifide.gradle.aem.instance.tasks
+
+import com.cognifide.gradle.aem.common.AemDefaultTask
+
+open class InstanceCreateAndUp : AemDefaultTask() {
```

Comment:
to avoid such strange name we could just do same as we have in environment plugin.  there is no `environmentCreate` because `environmentUp` covers such functionality. you could do same. if you will achieve same good / obvious semantic as we have before, I will be fine with that. in other words, `InstanceCreateAndUp` / loooks so strange and I cannot agree to have it. By incorporating create functionality into up, maybe we could have compromise here... what do you think?

Reply:
Yes, agree